### PR TITLE
SKALE-2302 Change tokens lock period

### PR DESCRIPTION
**Pull request by DimaStebaev**
Lock tokens after network launch for 90 days instead of 3 months

### CODE REVIEW

File: test/utils/time.ts [hunk 1]
```diff
@@ -50,3 +50,9 @@ export async function currentTime(web3: Web3) {
 }
 
 export const months = ["jan", "feb", "mar", "apr", "may", "jun", "jul", "aug", "sep", "oct", "nov", "dec"];
+
+export async function isLeapYear(web3: Web3) {
+    const timestamp = await currentTime(web3);
+    const now = new Date(timestamp * 1000);
+    return now.getFullYear() % 4 === 0;
```
Comment (yavrsky) [inline test/utils/time.ts:57]: (now.getFullYear() % 4 === 0  &&  now.getFullYear() % 100 !== 0) || now.getFullYear() % 400 === 0

**Review by yavrsky — DISMISSED** [review]

File: test/utils/time.ts (hunk 1 above)
Comment (DimaStebaev) [reply test/utils/time.ts:57]: Agree but %4 will work for next 80 years and looks simpler.

**Review by cstrangedk — APPROVED** [review]
lgtm!

**Review by payvint — APPROVED** [review]
That's cool!

**Pull request merged**